dvc/main.py

```diff
@@ -64,6 +65,15 @@ def main(argv=None):
                 "unicode is not supported in DVC for Python 2 "
                 "(end-of-life January 1, 2020), please upgrade to Python 3"
             )
+        elif isinstance(exc, OSError) and exc.errno == 24:
+            solution_hint = ""
+            if os.name != "nt":
+                solution_hint = (
+                    ", increase open file descriptors limit with `ulimit`"
+                )
```

Comment:
The command is not full and it is different for mac and linux 🙂Windows also has ways to increase ulimit. WIthout the hint this message is useless, as it just duplicates what the exception says.

Reply:
Also, might consider just refering to our doc (e.g. in user-guide) and explaining nicely what is going on and how to solve it on different systems and why increasing ulimit is better for dvc.